app/components/sidebar.tsx

```diff
@@ -128,6 +130,45 @@ function useDragSideBar() {
   };
 }
 
+function useGlobalShortcut() {
+  const chatStore = useChatStore();
+  const navigate = useNavigate();
+  const config = useAppConfig();
+
+  const handleMasks = async () => {
+    await register("CommandOrControl+Shift+M", () => {
+      if (config.dontShowMaskSplashScreen !== true) {
+        navigate(Path.NewChat, { state: { fromHome: true } });
+      } else {
+        navigate(Path.Masks, { state: { fromHome: true } });
+      }
+    });
+  };
+
+  const handleSettings = async () => {
+    await register("CommandOrControl+,", () => {
+      navigate(Path.Settings);
+    });
+  };
+
+  const handleNewChat = async () => {
+    await register("CommandOrControl+N", () => {
+      if (config.dontShowMaskSplashScreen) {
+        chatStore.newSession();
+        navigate(Path.Chat);
+      } else {
+        navigate(Path.NewChat);
+      }
+    });
+  };
+
+  useEffect(() => {
+    handleMasks();
+    handleSettings();
+    handleNewChat();
+  }, []);
```

Comment:
add this comment
`// eslint-disable-next-line react-hooks/exhaustive-deps`

for example:
```tsx
  useEffect(() => {
    handleMasks();
    handleSettings();
    handleNewChat();
    // eslint-disable-next-line react-hooks/exhaustive-deps
  }, []);
```

for ignore/bypass that `stupid react/nextjs` warning

Reply:
Thanks for your review! It might be refactored into a modular structure if desktop shortcut feature is accepted.

Reply:
@JasonEWNL cek new one

Reply:
> Thanks for your review! It might be refactored into a modular structure if desktop shortcut feature is accepted.

@JasonEWNL The desktop version is actually quite useful compared to the web version, which has limitations. For instance, you can't leverage certain features on the web version, but you can definitely leverage them on the desktop version. since it's safe through memory leaks and other